CMP1903_A1_2324/Die.cs

```diff
@@ -20,7 +20,8 @@ internal class Die
         //Property
         //rolled value property to store what the die rolled
         public int rolledvalue = 0;
-        //Methodi th
+
+        //Method
         public int Roll()
```

Comment:
No head comment describing what the method does

Reply:
comment added